ezbake/config/conf.d/auth.conf

```diff
@@ -1,39 +1,116 @@
 authorization: {
-    # TODO: Fill these in with real default rule definitions - SERVER-846.
-    # Putting just the ones for the certificate authority in for now since
-    # these are always evaluated by trapperkeeper-authorization and are
-    # needed to complete a successful agent run.
+    version: 1
     rules: [
         {
+            # Allow nodes to retrieve their own catalog
+            match-request: {
+                path: "^/puppet/v3/catalog/([^/]+)$"
+                type: regex
+                method: [get, post]
+            }
+            allow: "$1"
+            sort-order: 500
+            name: "puppetlabs catalog"
+        },
+        {
+            # Allow nodes to retrieve the certificate they requested earlier
+            match-request: {
+                path: "/puppet-ca/v1/certificate/"
+                type: path
+                method: get
+            }
+            allow-unauthenticated: true
+            sort-order: 500
+            name: "puppetlabs certificate"
+        },
+        {
+            # Allow all nodes to access the certificate revocation list
             match-request: {
                 path: "/puppet-ca/v1/certificate_revocation_list/ca"
                 type: path
                 method: get
             }
             allow: "*"
+            sort-order: 500
+            name: "puppetlabs crl"
         },
         {
+            # Allow nodes to request a new certificate
             match-request: {
-                path: "/puppet-ca/v1/certificate/ca"
+                path: "/puppet-ca/v1/certificate_request"
                 type: path
-                method: get
+                method: [get, put]
```

Comment:
That would definitely matter, wouldn't it? But yes, it has - you'll see the version bump change in `project.clj`